src/openrct2/rct12/RCT12.h

```diff
@@ -50,6 +50,15 @@ constexpr uint16_t const RCT12_MAX_HELICES = 31;
 
 constexpr uint8_t RCT12_BANNER_INDEX_NULL = std::numeric_limits<uint8_t>::max();
 
+constexpr uint16_t const RCT12_XY8_UNDEFINED = 0xFFFF;
+
+// Everything before this point has been researched
+#define RCT12_RESEARCHED_ITEMS_SEPARATOR 0xFFFFFFFF
```

Comment:
Could make those constexpr too

Reply:
I think you're looking at the wrong PR ;-)
The one about `constexpr`ing stuff is here: https://github.com/OpenRCT2/OpenRCT2/pull/10855